pkgs/by-name/ka/kata-runtime/package.nix

```
@@ -0,0 +1,104 @@
+# Derived from https://github.com/colemickens/nixpkgs-kubernetes
+{
+  buildGoModule,
+  fetchFromGitHub,
+  fetchzip,
+  lib,
+  qemu_kvm,
+  system,
+  virtiofsd,
+  yq-go,
+}:
+
+let
+  version = "3.7.0";
+
+  imageSuffix =
+    {
+      "x86_64-linux" = "amd64";
+      "aarch64-linux" = "arm64";
+    }
+    ."${system}" or (throw "Unsupported system: ${system}");
+
+  kata-images = fetchzip {
+    name = "kata-images-${version}";
+    url = "https://github.com/kata-containers/kata-containers/releases/download/${version}/kata-static-${version}-${imageSuffix}.tar.xz";
+    sha256 = "sha256-6ySKAqrbHDRgVlI7wm2p4Uw96ZMzUpP00liujxlruSM=";
+
+    postFetch = ''
+      mv $out/kata/share/kata-containers kata-containers
+      rm -r $out
+      mkdir -p $out/share
+      mv kata-containers $out/share/kata-containers
+    '';
+  };
+
+in
+
+buildGoModule rec {
+  pname = "kata-runtime";
+  inherit version;
+
+  # https://github.com/NixOS/nixpkgs/issues/25959
+  hardeningDisable = [ "fortify" ];
+
+  src = fetchFromGitHub {
+    owner = "kata-containers";
+    repo = "kata-containers";
+    rev = version;
+    sha256 = "sha256-Ir+/ZZJHm6E+044wczU3UvL+Py9Wprgw2QKJaYyDrKU=";
```

```suggestion
    hash = "sha256-Ir+/ZZJHm6E+044wczU3UvL+Py9Wprgw2QKJaYyDrKU=";
```
`sha256` is being phased out.

---

Done